specification/purview/resource-manager/readme.go.md

```diff
@@ -25,16 +25,6 @@ directive:
 ``` yaml $(go) && $(multiapi)
 batch:
   - tag: package-2021-07-01
-  - tag: package-2020-12-01-preview
```

Comment:
May I ask why we have to remove this package? Is this api-version no longer supported?

Reply:
We can keep it if that is the standard practice here, this was our public preview version while we are now going to GA

Reply:
Adding this back now, to avoid breakign change :)